web/src/services/UserInfo.ts

```diff
@@ -17,7 +17,11 @@ export interface MethodPreferencePayload {
     method: Method2FA;
 }
 
-export function toEnum(method: Method2FA): SecondFactorMethod {
+export function isMethod2FA(method: string) {
```

Comment:
Also don't see this as a changed file but maybe because of the rebase?

Reply:
I'm not entirely sure about this one, pretty sure I changed it though since there were multiple toEnum methods around and I neded to use both somewhere.